dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -0,0 +1,351 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign, ArticleSpecial } from '@guardian/libs';
+import {
+	between,
+	border,
+	from,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
+import { getSoleContributor } from '../lib/byline';
+import { palette as themePalette } from '../palette';
+import type { Branding as BrandingType } from '../types/branding';
+import type { TagType } from '../types/tag';
+import {
+	metaContainer as metaContainerMargins,
+	shouldShowAvatar,
+	shouldShowContributor,
+} from './ArticleMeta';
+import { Avatar } from './Avatar';
+import { Branding } from './Branding.importable';
+import { CommentCount } from './CommentCount.importable';
+import { Contributor } from './Contributor';
+import { Dateline } from './Dateline';
+import { FollowWrapper } from './FollowWrapper.importable';
+import { Island } from './Island';
+import { SendAMessage } from './SendAMessage.importable';
+
+type Props = {
+	format: ArticleFormat;
+	pageId: string;
+	webTitle: string;
+	byline?: string;
+	tags: TagType[];
+	primaryDateline: string;
+	secondaryDateline: string;
+	branding?: BrandingType;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	isCommentable: boolean;
+	ajaxUrl: string;
+	messageUs?: MessageUs;
+};
+
+const metaGridContainer = (showAvatar: boolean) => css`
+	display: grid;
+	grid-template-rows: auto;
+
+	${showAvatar
+		? css`
+				grid-template-columns: auto 1fr auto;
+				grid-template-areas:
+					'avatar byline comment-count'
+					'avatar byline comment-count'
+					'dateline dateline dateline';
+		  `
+		: css`
+				grid-template-columns: 1fr auto;
+				grid-template-areas:
+					'byline comment-count'
+					'byline comment-count'
+					'dateline dateline';
+		  `}
+`;
+
+const MetaGridAvatar = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: avatar;
+			width: 140px;
+			height: 140px;
+			margin-top: 6px;
+			margin-right: 10px;
+			margin-bottom: 12px;
+			margin-left: 0px;
+
+			${until.leftCol} {
+				grid-area: avatar;
+				width: 60px;
+				height: 60px;
+				margin-top: 3px;
+				margin-right: 10px;
+				margin-bottom: 12px;
+				margin-left: 0px;
```

Comment:
Not a blocker, but we generally favour mobile-first designs, so your top-level styles would be the mobile ones, and your media query would be `from.leftCol`.

---

Here `grid-area` is redundant, and values should ideally use source spaces:

```suggestion
				width: 60px;
				height: 60px;
				margin-top: ${space[1]}px;
				margin-right: ${space[2]}px;
				margin-bottom: ${space[3]}px;
				margin-left: 0px;
```

Reply:
You mean `from.desktop` no? I'm assuming mobile design only goes up to table size here.

Reply:
Is it even neccessary to support anything larger than tablet in the CSS for an Apps only component?